services/movie.py

```diff
@@ -1,14 +1,21 @@
+from typing import Optional
+
+from django.db import transaction
 from django.db.models import QuerySet
 
 from db.models import Movie
 
 
 def get_movies(
-    genres_ids: list[int] = None,
-    actors_ids: list[int] = None,
+        genres_ids: Optional[list[int]] = None,
```

Comment:
still overindented ( must be 4 spaces )

Reply:
> still overindented ( must be 4 spaces )

when I press enter to move attributes to new line it drops it to that place, I never did it manually, ctrl+alt+L also does same